.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,19 @@
+STOP! Are you reporting a bug, or something that seems bug-like? If
+so, please report under the ARROW project on the ASF JIRA server
+https://issues.apache.org/jira.
```

Comment:
How about using the ARROW project link here? https://issues.apache.org/jira/browse/ARROW

Might also be good to note that ASF JIRA is free and open to the public and anyone can sign themselves up for an account (that wasn't obvious to me before I was involved and isn't the case with bug trackers for some other projects.)

Reply:
done